src/config.rs

```diff
@@ -1,9 +1,28 @@
-use crate::errors::CargoMSRVError;
+use crate::errors::{CargoMSRVError, TResult};
 use clap::ArgMatches;
 use rust_releases::semver;
 use std::convert::TryFrom;
 use std::path::{Path, PathBuf};
 
+#[derive(Debug, Clone, Copy)]
+pub enum OutputFormat {
+    /// Progress bar rendered to stderr
+    Human,
+    /// Json status updates printed to stdout
+    Json,
+    /// No output -- meant to be used for testing
+    None,
+}
+
+/// Gets a [`Config`] from the given matches, but sets output_format to None
+///
+/// This is meant to be used for testing
+pub fn test_config_from_matches<'a>(matches: &'a ArgMatches<'a>) -> TResult<Config<'a>> {
```

Comment:
Since this is for testing, we can set the privacy to `pub(crate)` and add the `#[cfg(test)]` attribute to the function.

Reply:
I initially wanted to put that way as well, but currently only the integration tests use it and wouldn't have access to it if it were private.

Reply:
Ai, you're right!